jetty-nosql-kvs/src/main/java/org/eclipse/jetty/nosql/kvs/KeyValueStoreSessionIdManager.java

```diff
@@ -110,11 +110,9 @@ protected void doStop() throws Exception
      */
     public boolean idInUse(final String idInCluster)
     {
-        byte[] dummy = idInCluster.getBytes(); // dummy string for reserving key
-        boolean exists = !addKey(idInCluster, dummy);
+        return getKey(idInCluster) != null; // note "null" may also mean be caused by connection problem. Anyway: Treating this as "not in use"
```

Comment:
Checking the existence of key by getKey would cause race condition of the the session id. memcached's 'add' will set key only if the specified key doesn't exist.